### DISCUSSION TITLE
How to get the info of the Prometheus cluster

### DISCUSSION THREAD

**Discussion by gupf0719** [prometheus-operator/kube-prometheus#664] [General]
How to get the info of the prometheus cluster in prometheus operator?
such as , i have 2 replicas of prometheus in operator

i need the pod's ip of 2 replicas ,how can i get them ? in operator or by the API

**Comment by rajpratik71** [comment]
> How to get the info of the prometheus cluster in prometheus operator?
> such as , i have 2 replicas of prometheus in operator
> 
> i need the pod's ip of 2 replicas ,how can i get them ? in operator or by the API

`kubectl get svc --all-namespaces `

or,

`kubectl get all --all-namespaces`

will print a lot of detail.

If you still unable to find the detail , Use `-o wide ` in commands.
